src/core/issues-notes.js

```diff
@@ -23,6 +23,14 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
+    $ins = $ins.filter(function (index, item, array) {
```

Comment:
`array` is not used so it can be removed.

Reply:
yup I'll make these changes